### ISSUE TITLE
cli: add a 'stages' command

### ISSUE THREAD

**Issue by ivotron** [systemslab/popper#229]
Similar to the [`popper env`](https://github.com/systemslab/popper/blob/master/cli/popper/commands/cmd_env.py) command, for this issue we will add a new `popper stages` command that displays the stages defined in the `.popper.yml` file. This will also require to add a `--set` flag to re-define the stages of an existing pipeline.

**Comment by AnishHota** [comment]
Here,Do you propose to add another file cmd_stages.py which will carry out the tasks as defined?

**Comment by ivotron** [comment]
@AnishHota yes, every command has it's own `cmd_...py` file, so we will have a `cmd_stages.py` for this too.

**Comment by AnishHota** [comment]
@ivotron Okay, got it.Thanks.

**Comment by AnishHota** [comment]
@ivotron What do you mean by 'redefine the stages of an existing pipeline'?

**Comment by ivotron** [comment]
@AnishHota it means that when you define a pipeline, you do something like this:

```bash
popper init --stages=first,second,last mypipeline
```

And the above will result on a pipeline named `mypipeline` with those three stages. Let's say later I'd like to modify the stages, so I'd do:

```bash
popper stages  # this should show the current stages

popper stages --set=one,two,three # redefine which stages that pipeline has
```

One thing we can do is to check if files for the given stages exist. So in the last line from above, the `stages` command would check if there's a bash script named `one` or `one.sh`.

I hope that helps
